src/main/java/core/basesyntax/figures/Circle.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax.figures;
+
+import core.basesyntax.behavior.Shape;
+
+public class Circle implements Shape {
+    private String name;
+    private String color;
```

Comment:
you should create an abstract parent class Figure with fields name and color
and with a constructor that accepts name and color